### DISCUSSION TITLE
Merging of direct and extended eGRN

### DISCUSSION THREAD

**Discussion by r-trimbour** [aertslab/scenicplus#459] [Q&A]
Hi Scenicplus developers,

I'm currently working on GRN method comparison and would like to end up with a single eGRN for each method.
I wanted thus to ask you: would you recommend to merge direct and extended cistromes (right after `prepare_menr`), or later after extended and direct eGRN are calculated ?

I would much prefer do the merging at the cistrome step, but I was a bit worried that the final eGRN would be modified a lot compared to the 2 classically produced by the scenicplus pipeline, since TF-gene association might differ.
I would appreciate any recommendation/opinion on it.

Thanks a lot for this tool and your efforts to make it a nice modular snakemake pipeline :)
Rémi

**Comment by SeppeDeWinter** [comment]
Hi @r-trimbour 

We mostly split them because the extended might contain more false positives (we have actually never tested this).

Could you explain why you prefer to do the merging at the cistrome step? I would do the merging at the end of the pipeline, i.e., take the union across both the extended and direct eGRN.

I'm glad you like the snakemake pipeline!

I'm happy to have a voice call to discuss these things, also if you need further help with implementing SCENIC+ in the benchmark framework (that's actually the reason I made the snakemake pipeline in the first place).

All the best,

Seppe
